tf2onnx/onnx_opset/nn.py

```diff
@@ -311,8 +311,9 @@ def conv_kernel_shape(ctx, node, input_idx, spatial=2):
     # Get spatial part.
     kernel_shape = kernel_shape[:spatial]
 
-    # Set new value and return it.
-    node.set_attr("kernel_shape", kernel_shape)
+    # Set attribute value only if all dimensions are known.
+    if all(d > 1 for d in kernel_shape):
```

Comment:
Should it be "d > 0"?

Reply:
Yes, thanks, that should be `d > 0` or even `d >= 0`. Depending on answer to next comment.